Malmo/samples/Cpp_examples/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 # ------------------------------------------------------------------------------------------------
 
+cmake_minimum_required( VERSION 2.8 )
```

Comment:
Have just twigged that this CMakeLists.txt _isn't_ actually the user-facing one that gets installed to the cpp samples folder, and shouldn't actually be used stand-alone - it should always be called from the root CMakeLists.txt, so will inherit the root's cmake_minimum_required command. The stand-alone CMakeLists.txt that ends up in the samples folder is actually CMakeLists.txt.in, which already correctly contains a minimum_required command.